samples/extauthz/src/main.go

```diff
@@ -46,9 +47,10 @@ const (
 )
 
 var (
-	httpPort = flag.String("http", "8000", "HTTP server port")
-	grpcPort = flag.String("grpc", "9000", "gRPC server port")
-	denyBody = fmt.Sprintf("denied by ext_authz for not found header `%s: %s` in the request", checkHeader, allowedValue)
+	serviceAccount = flag.String("allow_service_account", "a", "allow service account extracted from source principle")
```

Comment:
can you update the usage a little bit: "allowed service account, matched against the service account in the source principal from the client certificate"